autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -52,15 +58,15 @@ if ($BuildOption -Eq "jackonwindows")
 {
     echo "Install JACK2 64-bit..."
     # Install JACK2 64-bit
-    choco install --no-progress -y jack
+    choco install --no-progress -y jack --version $JackVersion
```

Comment:
Why doesn’t the variable follow the style ${} here?

Reply:
Missed it... :)
Fixed, thanks.